knora/dsplib/utils/xml_upload.py

```diff
@@ -306,12 +314,18 @@ def xml_upload(input_file: str, server: str, user: str, password: str, imgdir: s
         quit(0)
 
     if sys.platform.startswith("darwin") or sys.platform.startswith("linux"):
-        save_location = f"{os.path.expanduser('~')}/.dsp-tools"
-    elif sys.platform.startswith("win"):
-        save_location = "."
+        save_location = Path.home() / Path(".dsp-tools")
```

Comment:
`Path.home()` works on all operating systems, so no need to check this anymore :)